src/components/field/index.js

```diff
@@ -243,8 +243,23 @@ export default class TextField extends PureComponent {
     startAnimation(labelAnimation, options)
   }
 
+  /*
+     If the input type is TextInputMasked,
+     the input ref is buried an extra layer deep.
+
+     Inception.
+  */
+  correctInputRef() {
+    let { current: input } = this.inputRef;
+    if (input.current) {
+      return input.current.input;
+    } else {
+      return input;
+    }
```

Comment:
I think this has to do with this code in the masking library:

```
ref={ref => {
        this.input = ref
        ...} }
```

So the text input ref is put under the "input". I don't think there's a way of resolving this outside of making changes to the masking library

Reply:
That's super bizarre - like they made their own kind of forward ref.